src/dbnode/storage/shard_test.go

```diff
@@ -811,7 +811,11 @@ func addTestSeries(shard *dbShard, id ident.ID) series.DatabaseSeries {
 }
 
 func addTestSeriesWithCount(shard *dbShard, id ident.ID, count int32) series.DatabaseSeries {
-	seriesEntry := series.NewDatabaseSeries(id, ident.Tags{}, 1, shard.seriesOpts)
+	seriesEntry := series.NewDatabaseSeries(series.DatabaseSeriesOptions{
```

Comment:
General style question: Should we keep `FooOptions` nomenclature specifically for interfaces with getters and setters, or just for collections of correlated values?

Reply:
My gut feel is to use it wherever it makes refactoring easy (look at how many call sites I had to update just to add an extra optional arg to `NewDatabaseSeries(...)`).